python/lsst/pipe/tasks/postprocess.py

```diff
@@ -607,19 +604,29 @@ class TransformObjectCatalogConfig(TransformCatalogBaseConfig):
         default="deep",
         doc="Name of coadd"
     )
+    # TODO: remove in DM-27177
     filterMap = pexConfig.DictField(
         keytype=str,
         itemtype=str,
         default={},
         doc=("Dictionary mapping full filter name to short one for column name munging."
              "These filters determine the output columns no matter what filters the "
-             "input data actually contain.")
+             "input data actually contain."),
+        deprecated=("Coadds are now identified by the band, so this transform is unused."
+                    "Will be removed after v22.")
+    )
+    outputBands = pexConfig.ListField(
+        dtype=str,
+        default=None,
```

Comment:
Once upon a time you had to set`optional=True` in order to have a config option be None.

Reply:
Oh, good point! None of the existing code actually uses this in a way that triggers `Config.validate()`, so it wasn't failing. I've added the `optional` now though.